yt/yt/tests/integration/node/test_user_job.py

```diff
@@ -55,19 +55,46 @@ def find_operation_by_mutation_id(mutation_id):
 ##################################################################
 
 
-class TestSandboxTmpfs(YTEnvSetup):
+class TestJobEnvironmentBase(YTEnvSetup):
     NUM_MASTERS = 1
     NUM_NODES = 1
     NUM_SCHEDULERS = 1
-    USE_PORTO = True
 
     DELTA_MASTER_CONFIG = {
         "cypress_manager": {
             "default_table_replication_factor": 1,
             "default_file_replication_factor": 1,
+        },
+        "chunk_manager": {
+            "allow_multiple_erasure_parts_per_node": True
         }
     }
 
+
+class TestPortoJobEnvironmentBase(TestJobEnvironmentBase):
+    USE_PORTO = True
+    # TODO(khlebnikov): For now tmpfs available only for porto.
+
+
+class TestCriJobEnvironmentBase(TestJobEnvironmentBase):
+    JOB_ENVIRONMENT_TYPE = 'cri'
+
+
+class TestEachJobEnvironmentBase(TestJobEnvironmentBase):
```

Comment:
I am not happy with this idea because class names are hidden within this code and it complicated discovery of the tests.

I think that more verbose scheme is actually better. Like this:
```python
class BaseTestJobInputCache(...):
    ...

class TestJobInputCachePorto(BaseTestJobInputCache):
     USE_PORTO = True

class TestJobInputCacheCri(BaseTestJobInputCache):
     JOB_ENVIRONMENT_TYPE = 'cri'

```

Reply:
Ok